build/build-runtimeimages.sh

```diff
@@ -36,23 +36,29 @@ function getTagName()
     return 0
 }
 
-
-echo
-echo "Generating Dockerfiles for Node runtime images..."
-$REPO_DIR/images/runtime/node/generateDockerfiles.sh
-
-echo
-echo "Generating Dockerfiles for .NET Core runtime images..."
-$REPO_DIR/images/runtime/dotnetcore/generateDockerfiles.sh
-
-echo
-echo "Generating Dockerfiles for Python runtime images..."
-$REPO_DIR/images/runtime/python/generateDockerfiles.sh
-
-echo
-echo "Generating Dockerfiles for PHP runtime images..."
-$REPO_DIR/images/runtime/php/generate-dockerfiles.sh
-
+runtimeImagesSourceDir="$RUNTIME_IMAGES_SRC_DIR"
```

Comment:
The changes here are to help during development to only build runtime images that a dev asks for rather than try building all the images